src/calibre/gui2/ui.py

```diff
@@ -10,15 +10,16 @@
 '''The main GUI'''
 
 import apsw
-import collections
 import errno
 import gc
 import os
 import re
 import sys
 import textwrap
 import time
-from collections import OrderedDict
+from collections import (
+    OrderedDict, deque
+)
```

Comment:
This import is short enough it should be online line rather than a multiline parenthesized import. See the style in the rest of this file (and elsewhere).